packages/flutter/lib/src/widgets/locale_query.dart

```diff
@@ -0,0 +1,33 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'framework.dart';
```

Comment:
Yes, and I'm concerned that there will be many more classes like LocaleQuery/MediaQuery in the widget hierarchy needed to publish various bits of context data to other widgets.

What's the plan for this in the framework?